src/include/btmem.h

```diff
@@ -719,6 +719,9 @@ struct __wt_page {
  */
 struct __wt_page_deleted {
 	uint64_t txnid;			/* Transaction ID */
+#if TIMESTAMP_SIZE > 0
```

Comment:
It's probably no prettier, but does it make sense to abstract these three lines into another macro `WT_DECL_TIMESTAMP`, possibly with a variable name for a parameter?